docs/data/data-grid/pivoting/pivoting.md

```diff
@@ -2,16 +2,170 @@
 title: Data Grid - Pivoting
 ---
 
-# Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')🚧
+# Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan') 🧪
 
-<p class="description">Rearrange columns and values to view data from multiple perspectives.</p>
+<p class="description">Rearrange rows and columns to view data from multiple perspectives.</p>
 
-:::warning
-This feature isn't available yet, but it is planned—you can 👍 upvote [this GitHub issue](https://github.com/mui/mui-x/issues/214) to help us prioritize it.
-Please don't hesitate to leave a comment there to describe your needs, especially if you have a use case we should address or you're facing specific pain points with your current solution.
-:::
+The Data Grid Premium's pivoting feature lets users transform the data in their grid by reorganizing rows and columns, creating dynamic cross-tabulations of data.
+This makes it possible to analyze data from different angles and gain insights that would be difficult to see in the default grid view.
 
-With the pivoting feature, users would be able to turn a column into a row, transforming the column's values into new columns.
+## Quick start
+
+Pivoting is enabled by default and can be accessed through the icon in the toolbar.
+In the demo below, the pivot panel is already open and some pivoting parameters have been set.
+Use the **Pivot** switch at the top of the panel to toggle pivoting off and on.
+You can drag and drop existing columns in the **Rows**, **Columns**, and **Values** dropdown menus to change how the data is pivoted.
+
+{{"demo": "GridPivotingMultipleValues.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Understanding pivoting
+
+Pivoting transforms your data by reorganizing it along three key dimensions: rows, columns, and values.
+
+### Non-pivoted data
+
+"Non-pivoted data" refers to the data as it's presented in the Data Grid before pivoting.
+The demo below shows the non-pivoted dataset used throughout this doc to illustrate key pivoting features.
+
+{{"demo": "GridNonPivoted.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Rows
+
+The **Rows** menu defines how the data will be grouped vertically after pivoting.
```

Comment:
I'm finding this a bit confusing. For example, the concept of "grouped vertically" is kinda hard to grasp like this.
Maybe what we need is really a picture mapping the data grid in pivoting state, showcasing what is rows, values and columns.
perhaps something along the lines:
<img width="735" alt="pivoting-explanation" src="https://github.com/user-attachments/assets/97ef9105-fd12-4b43-8009-8ee37962fc2a" />

Reply:
Added in https://deploy-preview-9877--material-ui-x.netlify.app/x/react-data-grid/pivoting/#rows